plugins/nvim-lsp/helpers.nix

```diff
@@ -10,7 +10,7 @@
     serverName ? name,
     package ? pkgs.${name},
     extraPackages ? {},
-    cmd ? (cfg: null),
+    cmd ? (package: null),
```

Comment:
Is there any reason you're changing it to `package` instead of `config`? The config seems more versatile

Reply:
I did that because I had an infinite recursion. But it might have been caused by something else...
Let me try again

Reply:
Oh yeah i can see why it happens, since `cmd` is itself part of cfg. Maybe you can pass in `cfg` without `cmd`?

Reply:
Ok it seems to work !